draft-ietf-ppm-dap.md

```diff
@@ -624,87 +622,77 @@ Helper:
 
 {{dap-topology}} illustrates which participants exchange HTTP messages. Arrows
 go from HTTP clients to HTTP servers. Note that some DAP participants may be
-clients sometimes but servers at other times. It is even possible for a single
-entity to embody multiple participants. For example, the Collector could also
-be one of the Aggregators.
+HTTP clients sometimes but HTTP servers at other times. It is even possible for
+a single entity to perform multiple DAP roles. For example, the Collector could
+also be one of the Aggregators.
 
 In the course of a measurement task ({{task-configuration}}), each Client
-records its own measurement value(s), packages them up into a report, and sends
-them to the Leader. Each share is encrypted to only one of the two Aggregators
-so that even though they pass through the Leader, the Leader is unable to see or
-modify the Helper's share. Depending on the task, the Client may only send one
-report or may send many reports over time.
+records its own measurement, packages it up into a report, and sends it to the
+Leader. Each share is encrypted to only one of the two Aggregators so that even
+though they pass through the Leader, the Leader is unable to see or modify the
+Helper's share. Depending on the task, the Client may only send one report or
+may send many reports over time.
 
 The Leader distributes the shares to the Helper and orchestrates the process of
-verifying them (see {{validating-inputs}}) and assembling them into an aggregate
-result for the Collector. Depending on the VDAF, it may be possible to
-incrementally process each report as it is uploaded, or it may be necessary to
-wait until the Collector transmits a collection request before processing can
-begin.
+verifying them and assembling them into a aggregate shares for the Collector.
+Depending on the VDAF, it may be possible to process each report as it is
+uploaded, or it may be necessary to wait until the Collector transmits a
+collection request before processing can begin.
```

Comment:
WDYT?
```suggestion
uploaded, or it may be necessary to wait until the Collector initializes a collection job before processing can begin.
```

Reply:
Sure, that's more consistent with how we discuss collection jobs elsewhere.